### PR TITLE
cathy/pageNotFoundAdjective

### LINKED ISSUE #9: 404 Page not Found adjective
p tags, look up how to change font-size on tailwind website!
image

### PR DESCRIPTION
**Pull request by catheiwu**
Issue #9
p tags, changed-size

### CODE REVIEW

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
just add a bunch of padding below adjective adjective adjective so it matches the figma more 
image

**Comment by catheiwu** [comment]
Screen Shot 2024-07-01 at 6 57 02 PM
fixed padding

File: src/components/NotFound.jsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ const NotFound = () => {
   return (
     <div className="bg-witl-purple-400 min-h-screen font-marcellus-sc">
       404 Page Not Found
+      <p
+        className="text-witl-white-100 text-4xl mb-8 flex flex-col items-center justify-center"
+        style={{ paddingBottom: "60px" }}
```
Comment (stanleylew5) [inline src/components/NotFound.jsx]: use tailwind class utility --- such as pb-12 or something like that find a value that works and matches the figma. pb stands for padding bottom. check the tailwind website for useful classes, don't use vanilla css

Comment (catheiwu) [reply src/components/NotFound.jsx]: made changes to use tailwind class utility

**Pull request merged**